ow_lander/src/ow_lander/actions.py

```diff
@@ -1227,7 +1227,7 @@ class PanTiltMoveJointsServer(mixins.PanTiltMoveMixin, ActionServerBase):
   def execute_action(self, goal):
```

Comment:
Line 1319 of this file needs the same change.

Reply:
That's more just a misleading line. It's actually is an `ArmExecutionAction` that it may catch, and that's only because the FrameMixin is typically only used by the arm, so it throws Arm errors. 

So that it doesn't confused readers of the code I went ahead and changed it to the base clase `ActionError`. I should probably clean-up the semantics of FrameMixin and make it more general, but that would affect so many arm actions that I would much rather not attempt such a change in a hotfix.